dhis2/dhis2-core
# feat: repair default icons job [DHIS2-18271]

### Summary
Adds a new job that runs every 10 minutes to repair default icons that have no file. The file is then re-stored in the file content store. 

Also adds an endpoint to manually invoke the same action using `PATCH /api/icons` (no body or parameters).

### Automatic Testing
A new controller test was added

### Manual Testing
For local disk storage backed file content store:
* goto DHIS2_HOME  `/files/icons` for the environment and remove some (or all) of the icon files
* start the server and wait until the job did run, you can check via `/api/jobConfigurations/IconsRepair/progress`
* check the deleted icons are restored (assuming they were default icons)
